### ISSUE TITLE
Should there be an InsertOrReplaceAllAsync to match InsertOrReplaceAsync 

### ISSUE THREAD

**Issue by b099l3** [praeclarum/sqlite-net#848]
Currently, there are methods for:
`Insert`
`InsertAsync`
`InsertAll`
`InsertAllAsync`
`InsertOrReplace`
`InsertOrReplaceAsync`

Feels like we are missing
`InsertOrReplaceAll`
`InsertOrReplaceAllAsync`

I know there are ways around this e.g. 
```csharp
InsertAllAsync(items, "OR REPLACE");
```
but I was thinking just to keep consistency to add the two new methods. I don't think it will be a big change from looking at the code. Just wanted to see if it was a good idea before starting work on it?

**Comment by blair0r** [comment]
Yes, please! That would be a great help to people just moving from the old SQLite.Net package to the new.

**Comment by K232** [comment]
Thanks @b099l3 , I wasn't aware of this hack! 👍  Thought I had to loop through all records to find out whether insert or update needs to be done.... Glad I found your post!! So yes, I would highly appreciate an InsertorReplaceAsync as I guess I'm not the only one .... 👍
